### PR TITLE
Feature/authentication

### PR DESCRIPTION
**Pull request by ioanagirbap33**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -5,17 +5,23 @@ import FeedbackForm from "./pages/feedback/FeedbackForm";
...
+  const { userData } = useAuth();
+
   return (
     <>
       <Router>
         <Header />
         <Routes>
           <Route path="/" element={<HomePage />} />
           <Route path="/movies/:id" element={<MoviePage />} />
+          {/* <Route path="/watchlist" element={<WatchListPage />} /> */}
```
Comment (roxanap33) [inline frontend/src/App.tsx]: Please delete this comment

File: frontend/src/components/button/Button.styled.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { rem } from "../../utils/rem";
 
 interface ButtonProps {
   type: "primary" | "text" | "secondary" | "icon" | "tertiary";
+  // isSubmit?: boolean;
```
Comment (adrianacostep33) [inline frontend/src/components/button/Button.styled.tsx]: Please delete this if you don't need it

File: frontend/src/components/input/Input.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import { rem } from "../../utils/rem";
+
+export const InputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: ${rem(20)} ${rem(0)};
+`;
```
Comment (adrianacostep33) [inline frontend/src/components/input/Input.styled.tsx:9]: Please add an empty line between components

File: frontend/src/components/input/Input.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  font-size: ${rem(18)};
+`;
+export const StyledInput = styled.input`
+  background: #575c7b;
+  padding: ${rem(8)};
+  border-radius: ${rem(8)};
+  height: ${rem(25)};
+  width: ${rem(250)};
+  color: #eaeaea;
+  @media (max-width: 500px) {
```
Comment (adrianacostep33) [inline frontend/src/components/input/Input.styled.tsx:23]: And before media query

**Review by adrianacostep33 — APPROVED** [review]

**Review by roxanap33 — APPROVED** [review]

**Pull request merged**